src/containers/search-bar/SearchBar.js

```diff
@@ -60,11 +61,12 @@ class SearchBar extends Component {
       this.setState({ value: '' })
       return
     }
+    const decodedValue = decodeURIComponent(value)
```

Comment:
we should make sure to test searches that intentionally include a % sign, etc. as well as tag search